scripts/after-pack-hook.js

```diff
@@ -44,4 +45,8 @@ module.exports = async function (params) {
   await fs.copy(distNodeModules, appNodeModules)
 
   console.log('all node_modules subfolders copied to', outputFolder)
+
+  if (process.env.DISABLE_SNAPSHOT_REQUIRE == null) {
```

Comment:
Feels odd that setting `DISABLE_SNAPSHOT_REQUIRE=0` prevents snapshots from running. Do our other env variables behave like this as well?

Reply:
Looks like we handle it multiple ways:

https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/cloud/api.ts#L105

https://github.com/cypress-io/cypress/blob/develop/system-tests/projects/react-app-webpack-5-unconfigured/config/jest/babelTransform.js#L6

i'm fine with making this more explicitly 1 or true maybe? I think eventually this environment variable will go away anyway